### DISCUSSION TITLE
Changing Theme in Rednotebook not working in newest update

### DISCUSSION THREAD

**Discussion by davidicharp40** [jendrikseipp/rednotebook#688] [General]
I noticed on the newest update there is no "etc/gtk 3.0" folder to change the theme in the settings.ini file. Where do I go about changing the settings to have the program change to a FlatStudioDark theme?

**Comment by jendrikseipp** [comment]
I don't know sorry. Maybe a Google search helps? Please post your results here for the benefit of others.
